pyincore/analyses/galvestoncge/galvestoncge.py

```diff
@@ -0,0 +1,2108 @@
+
+            return None
+
+        '''
+        Calibrate the model 
+        '''
+
+        soln = []
+        # TODO: I am not sure this is needed here. We might want to leave the python version
+        # of the models out for now
+        filename = "ipopt_cons.py"
+        tmp = "tmp.py"
+        print("Calibration: ")
+        run_solver(filename, tmp)
+
+        # sys.exit()
+        '''
+        Simulation code below:
+        In each simulation:
+
+        1. Apply simulation code (for instance PI(I) = 1.02).
+        2. Rewrite all equations
+        3. Solve the new model with the result from last run as initial guess.
+
+        '''
+
+        '''
+        ######## The following is for individual shocks. ######## 
+
+        iNum = 1 # dynamic model itterations
+
+        for ittr in range(iNum):
+          print("Simulation: ", ittr+1)
+          if ittr == 0: # if it is the first simulation, apply the shock
+
+              #DELTA.loc[I] = 1.02 * DELTA.loc[I]
+
+              KS0.loc[K, I] = KS0.loc[K, I]*0.7
+
+              #KS0.loc[K, ['HS1']] = KS0.loc[K, ['HS1']] * 0.675
+              #KS0.loc[K, ['HS2']] = KS0.loc[K, ['HS2']] * 0.739
+              #KS0.loc[K, ['HS3']] = KS0.loc[K, ['HS3']] * 0.958
+              #KS0.loc[K, ['GOODS']] = KS0.loc[K, ['GOODS']] * 0.658
+              #KS0.loc[K, ['TRADE']] = KS0.loc[K, ['TRADE']] * 0.961
+              #KS0.loc[K, ['OTHER']] = KS0.loc[K, ['OTHER']] * 0.673
+
+
+          else: # other simulations
+
+              #KS0 = KSNEW*(1-DEPR)+vars.get('N', x=soln[-1])
+              KS0 = vars.get('KS', x=soln[-1])
+
+          run_solver(filename, tmp)
+
+        '''
+
+        '''
+         ######## The following is for random shocks. ########  
+        '''
+        # iNum = 1 # dynamic model itterations
+        sims = sector_shocks
+        all_sectors = ['IAGMIN', 'IUTIL', 'ICONS', 'IMANU', 'IWHOLE', 'IRETAIL', 'ITRANS', 'IPROFSER', 'IREALE', 'IEDU',
+             'IHEALTH', 'IART', 'IACCO',
+             'MAGMIN', 'MUTIL', 'MCONS', 'MMANU', 'MWHOLE', 'MRETAIL', 'MTRANS', 'MPROFSER', 'MREALE', 'MEDU',
+             'MHEALTH', 'MART', 'MACCO',
+             'HS1I', 'HS2I', 'HS3I', 'HS1M', 'HS2M', 'HS3M']
+
+
+        missing_sectors = list(set(all_sectors) - set(sims.index.unique()))
+
+        for i in missing_sectors:
+            data = {'shock':1}
+            index = [i]
+            df = pd.DataFrame(data=data, index=index)
+
+            sims = sims.append(df)
+
+
+        # === begin replacing the average shocks on housing services
+        HSmean = sims.loc[['HS1I', 'HS2I', 'HS3I']].mean().mean()
+        sims.loc[['HS1I', 'HS2I', 'HS3I']] = HSmean
+        # === end replacing the average shocks on housing services
+
+        iNum = 1
+        # iNum = len(sims.columns)
+        KS00 = KS0.copy()
+
+        for num in range(iNum):
+            KS0.loc[K, I] = KS00.loc[K, I].mul(sims.iloc[:, num])
+            KS0 = KS0.fillna(0.0)
+            run_solver(filename, tmp)
```

Comment:
Could you catch infeasible and other wrong solver status here? Example you can see what Chris did for SLC
https://github.com/IN-CORE/pyincore/pull/290/files#diff-8047c3f02337ce7d619fd574adc25a744604ea540b0505a25957485c25199ae4R1868